src/database/connection.database.ts

```diff
@@ -1,12 +1,10 @@
 import knex from 'knex';
-import {Pool} from 'pg';
 import {config} from 'dotenv';
 
 config();
 
 export const connection: knex = knex({
   client: 'pg',
-  pool: {min: 10, max: 10000},
```

Comment:
The approach with db is faulty: a new connection is created for each query, which results in exceeding connection limit. To solve it completely, a lot of code should be rewritten, but the worst thing comes from the pool. Removing the pool (together with increasing db connection limit in our e2e compose file) make the problem unnoticeable (at least for our e2e test usage)

Reply:
Nice. It would be best to create an issue in the [ArweaveTeam/gateway](https://github.com/ArweaveTeam/gateway) project. This way we can make sure we got it right.